packages/BarChart/src/__snapshots__/BarChart.test.tsx.snap

```diff
@@ -20,7 +20,7 @@ exports[`BarChart It should render a snapshot 1`] = `
         <div
           class="ids-bar-chart__graph"
           data-value="48"
-          style="width: 100%; background: rgb(251, 199, 59);"
+          style="width: 0px; background-color: rgb(251, 199, 59);"
```

Comment:
px shouldn't be needed when 0. I'd suggest removing it.

Reply:
the value is setter by `react-spring`

Reply:
oh right, it's the snapshot, sorry